test/dotnet.Tests/dotnet.Tests.csproj

```diff
@@ -23,47 +21,49 @@
       <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
     </Content>
   </ItemGroup>
-  
-  <Target Name="WriteExpectedVersion"
-          BeforeTargets="BeforeBuild"
-          DependsOnTargets="PrepareForBuild">
-
+  <Target Name="WriteExpectedVersion" BeforeTargets="BeforeBuild" DependsOnTargets="PrepareForBuild">
```

Comment:
Why change the format here?

Reply:
None of this stuff actually changed here, just formatting.

Reply:
(VS change it like this every time...)

Reply:
Will revert

Reply:
Fixed